core/storage/feedback/gae_models.py

```diff
@@ -623,14 +508,15 @@ def create(
 
         Raises:
             Exception: There is already a feedback thread with the same
-                exploration_id and thread_id.
+                thread_id.
         """
-        instance_id = cls._get_instance_id(exploration_id, thread_id)
+        instance_id = thread_id
         if cls.get_by_id(instance_id):
             raise Exception('There is already a feedback thread with the given '
                             'thread id: %s' % instance_id)
         state_content = cls._convert_suggestion_html_to_legacy_state_content(
             suggestion_html)
+        exploration_id = thread_id.split('.')[0]
```

Comment:
Doesn't generalize; we might need a plan for this if it involves a subsequent model migration. I'm fine with staging that to a later PR but a first step is getting all the exploration-specific logic for ID construction centralized in one place.

Reply:
Done